<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWindow</class>
 <widget class="QDialog" name="FindWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>Enter your criteria</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="position_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Experiense</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="experiense_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Salary</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="salary_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>401</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="view_vacancy_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View vacancy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
